build.gradle

```diff
@@ -46,6 +46,11 @@ task clean (type: GradleBuild) {
   tasks = ['clean']
 }
 
+task realclean (type: GradleBuild) {
```

Comment:
Usually 'clean' deletes the build artifacts while 'realclean' deletes everything that is not in the distributed software.